src/ChangeHighlight.tsx

```diff
@@ -31,12 +33,12 @@ const ChangeHighlight: React.FC<Props> = ({
 }) => {
   const [shadowDOM, setShadowDOM] = useState([]);
   const isInitialMount = useRef(true);
-  const [uniqueId, setUniqueId] = useState(0);
+  const [uniqueId, setUniqueId] = useState("");
 
   useEffect(() => {
     if (!disabled) {
       if (isInitialMount.current) {
-        setUniqueId(Math.floor(Math.random() * 100000)+1);
+        setUniqueId(generateUUID());
```

Comment:
I prefer to change the function name to have the proper verb, so it's more convenient to have `get` in the beginning, something like this `getUUID()`